src/components/photos/PhotoGalleryView.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import { Pressable, StyleSheet, View, Text, Dimensions } from "react-native";
+import { Icon, Image } from "react-native-elements";
+import ImageView from "react-native-image-viewing";
+import { Colors } from "../../utils/Colors";
+import { Spacing } from "../../utils/Spacing";
+
+interface Props {
+    source: string;
+}
+
+const PhotoGalleryView: React.FC<Props> = ({ source }) => {
+    const [liked, setLiked] = useState(false);
+    const [isViewing, setIsViewing] = useState(false);
+
+    return (
+        <>
+            <View>
+                <View>
+                    <Image
+                        source={{ uri: source }}
+                        style={styles.image}
+                        resizeMethod="resize"
+                    />
+                    <Pressable
+                        style={({ pressed }) => [
+                            {
+                                backgroundColor: pressed
+                                    ? Colors.TOUCH_HIGHLIGHT
+                                    : Colors.TRANSPARENT,
+                            },
+                            styles.image,
+                            styles.imageOverlay,
+                        ]}
+                        onPress={() => {
+                            setIsViewing(true);
+                        }}
+                    ></Pressable>
```

Comment:
Since there's nothing inside the `Pressable`, it can be changed to `/>` instead of `></Pressable>` I believe. Test it out and see if it works, otherwise ignore this.
```suggestion
                    />
```